.buildkite/pipeline.full.yml

```diff
@@ -272,6 +284,10 @@ steps:
     concurrency: 9
     concurrency_group: browserstack-app
     concurrency_method: eager
+    retry:
+      automatic:
+        - exit_status: "*"
+          limit: 1
```

Comment:
I wouldn't let test steps retry automatically, as it could mask flakes that need investigations.  Retries for things within the step that we're happy to be retried (e.g. app uploads) is the safest approach.

Reply:
...retrying build steps seems reasonable, as we can't control any flakiness in the Unity build system.

Reply:
That makes sense